config.sh

```diff
@@ -51,23 +51,27 @@ declare -a APT_GET_APPS=(
 )
 
 declare -a BREW_CASK_APPS=(
```

Comment:
Maybe this BREW_CASK_APPS change should be moved to your forked repository? I agree that maybe not everyone is going to use alfred, but some decisions are very subjective. Not sure, maybe I need to rethink this a little bit